pkg/services/thumbs/dashboardthumbsimpl/xorm_store.go

```diff
@@ -107,22 +107,22 @@ func (ss *xormStore) UpdateState(ctx context.Context, cmd *thumbs.UpdateThumbnai
 	return err
 }
 
-func (ss *xormStore) Count(ctx context.Context, cmd *thumbs.FindDashboardThumbnailCountCommand) (int64, error) {
-	err := ss.db.WithDbSession(ctx, func(sess *db.Session) error {
+func (ss *xormStore) Count(ctx context.Context, cmd *thumbs.FindDashboardThumbnailCountCommand) (n int64, err error) {
+	err = ss.db.WithDbSession(ctx, func(sess *db.Session) error {
```

Comment:
I have a similar comment here as the previous one. Could we set n directly? A similar question applies for a few of the other changes.